src/Data/Fin/Induction.agda

```diff
@@ -59,6 +60,20 @@ open WF public using (Acc; acc)
   induct {suc i} (acc rec) = Pᵢ⇒Pᵢ₊₁ i (induct (rec (inject₁ i) i<i+1))
     where i<i+1 = ℕ<⇒inject₁< (i<1+i i)
 
+≤-Induction : ∀ (P : Pred (Fin (suc n)) ℓ) {i} →
```

Comment:
Okay so we should come up with a good name for this lemma! It's weak induction from an arbitrary starting point. The obvious `<-weakInduction-arbitraryStart` is a bit of a mouthful though? Do you have any better ideas?

Reply:
I modified the name, but I didn't think about a better name.

Reply:
Okay, I've settled on `<-weakInduction-startingFrom` and have made the change myself.